scripts/buildDictionary.js

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+require('isomorphic-fetch');
+
+const fs = require('fs');
+const path = require('path');
+const pathExists = require('path-exists');
+const hash = require('object-hash');
+const ora = require('ora');
+const GwentAPI = require('gwent-api-client').default;
+
+const OUT_DIR = path.join(__dirname, '../dist/website');
+const CARDS_PATH = path.join(OUT_DIR, './cards.json');
+const VERSION_PATH = path.join(OUT_DIR, './version.json');
+const spinner = ora().start();
+
+Promise.resolve(pathExists(OUT_DIR))
+    .then(exists => {
+        if (!exists) {
+            throw new Error(
+                'The website must be built before creating the dump, run `yarn run build:website`'
+            );
+        }
+    })
+    .then(() => {
+        spinner.text = 'Fetching cards from the Gwent API...';
+        return GwentAPI.cards({ offset: 0, limit: 9999 });
+    })
+    .then(({ results }) => results)
+    .then(cards =>
+        cards.reduce(
+            (promise, card, index) =>
+                promise.then(acc =>
+                    GwentAPI.one(card, {
+                        fields: ['variations']
+                    }).then(fullCard => {
+                        spinner.text = `Downloading the cards: ${index}/${cards.length}`;
+                        return acc.concat([fullCard]);
+                    })
+                ),
+            Promise.resolve([])
+        )
+    )
+    .then(cards => {
+        spinner.text = 'Reducing cards disk size...';
```

Comment:
Ideally, we should also serve the gzipped version of cards ^^ but it's not possible using gh-pages.